### PR TITLE
File Added : All file changes done.

### PR DESCRIPTION
**Pull request by yogesh141293**

### CODE REVIEW

**Review by Oshin-250197 — APPROVED** [review]

**Review by SamikshaWani — APPROVED** [review]

File: Yogesh/HTML/Assig.1/Que1.html [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+    <meta name='viewport' content='width=device-width, initial-scale=1'>
+    <link rel='stylesheet' type='text/css' media='screen' href='main.css'>
+    <script src='main.js'></script>
+</head>
+<body>
+    <h1>HTML</h1>
+    <h3>Introduction</h3>
+    <p>HTML is Standard markup language for Web Pages. </p>
+</body>
+</html>
```
Comment (Pawan1005) [inline Yogesh/HTML/Assig.1/Que1.html]: need indentation
add new line at the end of file

File: Yogesh/HTML/Assig.1/Que3.html [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+    <h2>Introduction</h2>
+    <p>HTML is a standard markup language for Web Pages.</p>
+    <hr>
+    <h3>HTML Editor</h3>
+    <h4>Notepad or TextEdit.</h4>
+    <p>Web Pages can be created and modified by using professional HTML editors.<br>
+        However, for learning HTML we recommend a simple text editor like Notepad(PC) or TextEdit(Mac).
+    </p>
+</body>
+</html>
```
Comment (Pawan1005) [inline Yogesh/HTML/Assig.1/Que3.html]: need indentation
add new line at the end of file

File: Yogesh/HTML/Assig.1/Que4.html [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+    <title>Introduction Page</title>
+</head>
+<style>
+    body{
+        text-align: center;
+        color: white;
+        background-color: black;
+        
+    }
+</style>
```
Comment (Pawan1005) [inline Yogesh/HTML/Assig.1/Que4.html]: this needs to be in head tag

File: Yogesh/HTML/Assig.1/Que4.html [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+<body>
+    <h1 style="font-size: 70px; font-family: cursive;">Yogesh Sureshrao Ghodmare</h1>
+    <p style="font-size: 20px; padding: 15px;"> I am Yogesh, I am from Arvi,Wardha.Maharashtra.<br>
+        I completed my PG in Comp Sci in Nagpur University,.<br> I have 2yrs Exp in DDU-GKY Project as IT Trainer.<br>
+        My hobbies are Tavelling and Playing Cricket. <br>
+        I'm passionate about SocialWork.
+    </p>
+
+</body>
+</html>
```
Comment (Pawan1005) [inline Yogesh/HTML/Assig.1/Que4.html]: need indentation
add new line at the end of file

File: Yogesh_Ghodmare/HTML/Assig.1/Que1.html [hunk 5]
```diff
@@ -0,0 +1,17 @@
...
+<html>
+  <head>
+    <meta charset="utf-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <title>Question number 1</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <link rel="stylesheet" type="text/css" media="screen" href="main.css" />
+    <script src="main.js"></script>
+  </head>
+
```
Comment (Pawan1005) [inline Yogesh_Ghodmare/HTML/Assig.1/Que1.html]: remove extra line

File: Yogesh_Ghodmare/HTML/Assig.1/Que1.html [hunk 6]
```diff
@@ -0,0 +1,17 @@
...
+  <body>
+    <h1>HTML</h1>
+    <h3>Introduction</h3>
+    <p>HTML is Standard markup language for Web Pages.</p>
+  </body>
+</html>
```
Comment (Pawan1005) [inline Yogesh_Ghodmare/HTML/Assig.1/Que1.html:16]: need indentation

File: Yogesh_Ghodmare/HTML/Assig.1/Que2.html [hunk 7]
```diff
@@ -0,0 +1,22 @@
...
+    <p>HTML is a standard markup language for Web Pages.</p>
+    <h3>HTML Editor</h3>
+    <h4>Notepad or TextEditor</h4>
+    <p>
+      Web Pages can be created and modified by using profesion HTML editors.<br />
+      However, for learning HTML we recommend a simple text editor like
+      Notepad(PC) or TextEdit(Mac).
+    </p>
+  </body>
+</html>
```
Comment (Pawan1005) [inline Yogesh_Ghodmare/HTML/Assig.1/Que2.html:20]: need indentation
remove extra line

File: Yogesh_Ghodmare/HTML/Assig.1/Que3.html [hunk 8]
```diff
@@ -0,0 +1,24 @@
...
+    <h3>HTML Editor</h3>
+    <h4>Notepad or TextEdit.</h4>
+    <p>
+      Web Pages can be created and modified by using professional HTML
+      editors.<br />
+      However, for learning HTML we recommend a simple text editor like
+      Notepad(PC) or TextEdit(Mac).
+    </p>
+  </body>
+</html>
```
Comment (Pawan1005) [inline Yogesh_Ghodmare/HTML/Assig.1/Que3.html:21]: need indentation
remove extra line

File: Yogesh_Ghodmare/HTML/Assig.1/Que4.html [hunk 9]
```diff
@@ -0,0 +1,29 @@
...
+    <title>Introduction Page</title>
+  </head>
+  <style>
+    body {
+      text-align: center;
+      color: white;
+      background-color: black;
+    }
+  </style>
+
```
Comment (Pawan1005) [inline Yogesh_Ghodmare/HTML/Assig.1/Que4.html]: this needs to be head tag

File: Yogesh_Ghodmare/HTML/Assig.1/Que4.html [hunk 10]
```diff
@@ -0,0 +1,29 @@
...
+    </h1>
+    <p style="font-size: 20px; padding: 15px">
+      I am Yogesh, I am from Arvi,Wardha.Maharashtra.<br />
+      I completed my PG in Comp Sci in Nagpur University,.<br />
+      I have 2yrs Exp in DDU-GKY Project as IT Trainer.<br />
+      My hobbies are Tavelling and Playing Cricket. <br />
+      I'm passionate about SocialWork.
+    </p>
+  </body>
+</html>
```
Comment (Pawan1005) [inline Yogesh_Ghodmare/HTML/Assig.1/Que4.html:26]: need indentation
remove extra line

File: Yogesh_Ghodmare/HTML/Assig2/.vscode/settings.json [hunk 11]
```diff
@@ -0,0 +1,3 @@
+{
+    "liveServer.settings.port": 5501
+}
```
Comment (Pawan1005) [inline Yogesh_Ghodmare/HTML/Assig2/.vscode/settings.json]: remove this file

File: Yogesh_Ghodmare/HTML/Assig2/Que.1.html [hunk 12]
```diff
@@ -0,0 +1,63 @@
...
+    body {
+      background-color: mediumaquamarine;
+    }
+
+    h1 {
+      color: red;
+      text-align: center;
+    }
+  </style>
+
```
Comment (Pawan1005) [inline Yogesh_Ghodmare/HTML/Assig2/Que.1.html]: this needs to be in head tag

File: Yogesh_Ghodmare/HTML/Assig2/Que.1.html [hunk 13]
```diff
@@ -0,0 +1,63 @@
...
+        text-align: center;
+        color: black;
+        font-weight: bold;
+        font-size: 20px;
+      "
+    >
+      Thank you!!!
+    </p>
+  </body>
+</html>
```
Comment (Pawan1005) [inline Yogesh_Ghodmare/HTML/Assig2/Que.1.html:46]: need indentation
remove extra line

**Review by Pawan1005 — CHANGES_REQUESTED** [review]
need indentation
remove extra line  
put style tag in head tag in every file

**Review by Pawan1005 — APPROVED** [review]

**Pull request merged**